config/rca.conf

```diff
@@ -55,8 +55,8 @@
     //hot shard rca
     "hot-shard-rca": {
       "cpu-utilization" : 0.01,
-      "io-total-throughput-in-bytes" : 250000.0,
-      "io-total-syscallrate-per-second" : 0.1
+      "heap-alloc-rate-in-bytes" : 250000.0,
```

Comment:
`DEFAULT_HEAP_ALLOC_RATE_THRESHOLD_IN_BYTE_PER_SEC` in [HotShardRcaConfig.java](https://github.com/opensearch-project/performance-analyzer-rca/pull/295/files#diff-bdea14e4f0e0c8149675e5c940041f89de9ebe30f840ad596b7bb14da9f559a1) is `2500.0` while here it is `250000.0`. Can you update the value to be consistent ?